src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -176,8 +176,9 @@ export default {
 
 		autoComplete: {
 			type: Function,
-			required: true,
+			default: function(){ return [] },
```

Comment:
Make eslint happy:
```suggestion
			default: () => [],
```

Reply:
Wouldn't that exectue and give the array instead of the function? 🤔

Reply:
Seems to work  👍

Reply:
Same as your code, but using the function shorthand.
Only for `type = Object` or `Array` the default value function is executed.

Reply:
Ok. Didn't know about the different behaviour depending on the type. Thanks! 👍